lib/PropertiesPanel.js

```diff
@@ -1121,7 +1121,7 @@ PropertiesPanel.prototype._createPanel = function(element, tabs) {
         }
       });
 
-      if (self.config.customOptions) {
+      if (self.config.customOptions.groups) {
```

Comment:
Add (self.config.customOptions && self.config.customOptions.groups) in case customOptions isn't set.